elephant/spike_train_correlation.py

```diff
@@ -109,6 +110,12 @@ def corrcoef(binned_sts, binary=False):
         are counted as 1, resulting in binary binned vectors :math:`b_i`. If
         False, the binned vectors :math:`b_i` contain the spike counts per bin.
         Default: False
+    with_nans : bool, optional
+        If True, correlations of empty spike trains are given NaN values. If
+        False, a boolean array indicating empty (False) and non-empty (True)
+        spike trains is returned. In each case a warning is raised when empty
```

Comment:
Here, I would indicate that the additional array of booleans is returned in addition, e.g.:
"If False, a boolean array indicating empty (False) and non-empty (True) spike trains is returned as a second return value."